osh/client/osh-cli

```diff
@@ -36,9 +36,11 @@ def handle_deprecated_options(conf):
               file=sys.stderr)
 
     # xmlrpclib/httplib do not follow HTTP redirect
-    if "cov01.lab.eng.brq.redhat.com" in conf["HUB_URL"]:
-        conf["HUB_URL"] = "https://cov01.lab.eng.brq2.redhat.com/osh/xmlrpc"
-        print_warning("HUB_URL")
+    hosts = ("cov01.lab.eng.brq.redhat.com", "cov01.lab.eng.brq2.redhat.com", "osh.prodsec.redhat.com")
```

Comment:
The reason for this hard-coded transformation is that `client.conf` is packaged as `%config(noreplace)`: https://github.com/openscanhub/openscanhub/blob/55f0214d2656a1e7a74dd65c18520dbe1d350a90/osh.spec#L221

This means that users who changed the `client.conf` file locally will not start using the updated `HUB_URL` after installing the updated RPM package until they merge the configuration file changes manually.  Therefore, I would not put `osh.prodsec.redhat.com` into this tuple.  It is unlikely that users would set such a `HUB_URL` manually in their configuration file and, if they do, they should use a correct URL with no need to rewrite it.

Reply:
Makes sense, removed from tuple